packages/module/patternfly-docs/content/extensions/component-groups/examples/ShortcutTable/Shortcut.md

```diff
@@ -0,0 +1,40 @@
+---
+# Sidenav top-level section
+# should be the same for all markdown files
+section: extensions
+subsection: Component groups
+# Sidenav secondary level section
+# should be the same for all markdown files
+id: Shortcut
+# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
+source: react
+# If you use typescript, the name of the interface to display props for
+# These are found through the sourceProps function provided in patternfly-docs.source.js
+propComponents: ['Shortcut']
```

Comment:
Documentation  is missing props you need to export the ShortcutProps and pass them in to the function component specifying the prop types.